pyccl/cosmology.py

```diff
@@ -228,6 +238,12 @@ def __init__(
             self.nl_pk_emu = matter_power_spectrum
             matter_power_spectrum = 'emulator'
 
+        self.baryons = baryonic_effects
+        if not isinstance(self.baryons, baryons.Baryons):
+            if self.baryons not in (None, 'bcm'):
```

Comment:
Do we want to merge the Vd19 model first and add it to this PR?

Reply:
It wouldn't change anything in this PR, I think, right? I don't think we should support more than a single default string option ('bcm'). Anything else and the Cosmology initialiser will again become unwieldy.

Reply:
The change in the previous comment takes care of this